src/masonite/utils/helpers.py

```diff
@@ -520,6 +524,20 @@ def info(self, message):
         return self.success(message)
 
 
+def random_string(length=4):
+    """Generate a random string based on the length given.
+
+    Keyword Arguments:
+        length {int} -- The amount of the characters to generate (default: {4})
+
+    Returns:
+        string
+    """
+    return "".join(
+        random.choice(string.ascii_uppercase + string.digits) for _ in range(length)
```

Comment:
Should we use a constant time generation method for better security here ? Or is it not useful ?